install-app.sh

```
@@ -5,16 +5,17 @@
 # runs scripts under root user. 
 
 # install system updates and isntall python3-pip package using apt. '-yq' flags are 
-# used to suppress any interactive prompts - we won't be able to confirm operation 
-# when running the script as VM extention.  
+# used to suppress any interactive prompts - we won't be able to confirm operation
+# when running the script as VM extention.
```

The comment text here is fine but note the script should explicitly enable fail-fast behavior (add `set -euo pipefail` near the top) so errors cause the extension to fail early and are visible in the VM extension logs. This ties to the requirement that the CustomScript extension must reliably run install-app.sh (checklist item 3.11).